src/recovery/interface.cairo

```diff
@@ -57,7 +57,7 @@ enum EscapeStatus {
 /// @param ready_at when the escape can be completed
 /// @param target_signers the signers to escape
 /// @param new_signers the new signers to be set after the security period
-#[derive(Drop, Serde, starknet::StorePacking)]
```

Comment:
Is this default now?

Reply:
No, I thought it's not supported any more with the general changes to the storage mechanism. It turns out the "packing" trait can still be implemented. This derive stopped working but I haven't found anything on why that's the case. I'll try to resolve it quickly.